### PR TITLE
Update redis service docs

### PR DESCRIPTION
**Pull request by sptramer**

### CODE REVIEW

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis.md](https://github.com/sptramer/azure-docs-sdk-python/blob/update-redis/docs-ref-services/redis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707141948277237-62%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/redis.md [hunk 1]
```diff
@@ -47,4 +52,4 @@ redis_cache = redis_client.redis.create_or_update(
 # redis_cache is a RedisResourceWithAccessKey instance
 ```
 
-Explore more [sample Python code](https://azure.microsoft.com/resources/samples/?platform=python) you can use in your apps.
\ No newline at end of file
+Explore more [example Python code](https://azure.microsoft.com/resources/samples/?platform=python) you can use in your apps.
```
Comment (lisawong19) [inline docs-ref-services/redis.md]: There aren't any redis samples so you can remove this line

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis.md](https://github.com/sptramer/azure-docs-sdk-python/blob/update-redis/docs-ref-services/redis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707142018101117-62%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/redis.md [hunk 2]
```diff
@@ -21,13 +21,18 @@ Redis is an advanced key-value store, where keys can contain data structures suc
...
+
+Create and manage your Redis resources in your subscription with the Redis management API.
 
 ```bash
 pip install azure-mgmt-redis
 ```
 
-## Example
+[!div class="nextstepaction"]
+[Explore the Management APIs](/python/api/azure.mgmt.sql)
```
Comment (lmazuel) [inline docs-ref-services/redis.md]: azure.mgmt.redis

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/redis.md](https://github.com/sptramer/azure-docs-sdk-python/blob/update-redis/docs-ref-services/redis.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707182253524228-62%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**